attestation-agent/attester/src/bin/evidence_getter.rs

```diff
@@ -3,7 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-use attester::*;
+use attester::Attesters;
```

Comment:
```suggestion
use attester::CompositeAttester;
```

same change needed below